src/components/Budget/constants.js

```diff
@@ -8,12 +8,7 @@ export const BUDGET_STATUSES = {
   PLANNED: 'Planned',
 };
 
-export const BUDGET_STATUSES_OPTIONS = [
-  { label: 'ui-finance.budget.status.active', value: BUDGET_STATUSES.ACTIVE },
-  { label: 'ui-finance.budget.status.closed', value: BUDGET_STATUSES.CLOSED },
-  { label: 'ui-finance.budget.status.frozen', value: BUDGET_STATUSES.FROZEN },
-  { label: 'ui-finance.budget.status.planned', value: BUDGET_STATUSES.PLANNED },
-];
+export const BUDGET_STATUSES_OPTIONS = Object.values(BUDGET_STATUSES).map(status => ({ label: status, value: status }));
```

Comment:
but label shouldn't be the same as value. It should be translated. There is only way - to use `labelId`. So I suppose the proper fix would be to change `label` to `labelId` above

Reply:
Right. I didn't update my local `node_modules` and used non-actual code from `stripes-acq-components`